ddl-introduction.md

```diff
@@ -0,0 +1,183 @@
+
+Taking `ADD INDEX` as an example, the entire process of state change is as follows:
+
+```
+absent -> delete only -> write only -> write reorg -> public
+```
+
+For users, the newly created index is unavailable before the `public` state.
+
+<SimpleTab>
+<div label="Online DDL asychronous change before TiDB v6.2">
+
+Before v6.2.0, the process of handling asynchronous schema changes in the TiDB SQL layer is as follows:
+
+1. MySQL Client sends a DDL request to the TiDB server.
+
+2. After receiving the request, a TiDB server parses and optimizes the request at the MySQL Protocol layer, and then sends it to the TiDB SQL layer for execution.
+
+    Once the SQL layer of TiDB receives the DDL request, it starts the `start job` module to encapsulate the request into a specific DDL Job (that is, a DDL task), and then stores this Job in the corresponding DDL Job queue in the KV layer based on the statement type. The corresponding worker is notified of the Job that requires processing.
+
+3. When receiving the notification to process the Job, the worker determines whether it is the role of the DDL Owner. If it is the Owner role, it directly processes the Job. Otherwise, it exits without performing any processing.
+
+    If a TiDB server is not the Owner role, then another node must be the Owner. The worker of the node in the Owner role periodically checks whether there is an available Job that can be executed. If such a Job is identified, the worker will process the Job.
+
+4. After the worker processes the Job, it removes the Job from the Job queue in the KV layer and places it in the `job history queue`. The `start job` module that encapsulated the Job will periodically check the ID of the Job in the `job history queue` to see whether it has been processed. If so, the entire DDL operation corresponding to the Job ends.
+
+5. TiDB server returns the DDL processing result to the MySQL Client.
+
+Before TiDB v6.2.0, the DDL execution framework had the following limitations:
+
+- The TiKV cluster only has two queues: `general job queue` and `add index job queue`, which handle logical DDL and physical DDL, respectively.
+- The DDL Owner always processes DDL Jobs in a first-in-first-out way.
+- The DDL Owner can only execute one DDL task of the same type (logical or physical) at a time, which is relatively strict.
+
+These limitations might lead to some "unintended" DDL blocking behavior. For more details, see [SQL FAQ - DDL Execution](/faq/sql-faq.md#ddl-execution).
+
+</div>
+<div label="Parallel DDL framework starting from v6.2">
+
+Before TiDB v6.2, because the Owner can only execute one DDL task of the same type (logical or physical) at a time, which is relatively strict, and affects the user experience.
```

Comment:
```suggestion
Before TiDB v6.2.0, because the Owner can only execute one DDL task of the same type (either logical or physical) at a time, which is relatively strict, and affects the user experience.
```